### DISCUSSION TITLE
اضافه شدن تیکه کدهای جالب

### DISCUSSION THREAD

**Discussion by Mohammadgeek** [softwaretalks/newsletter#570] [Ideas]
سلام به نظرم رسید یکسری کدها جالب که به ذهنمون میرسه بزارم در قالب tricks میتونه جالب باشه یک قسمت مجزا مثل github gist

**Comment by ohmydevops** [comment]
سلام وقت بخیر
من متوجه منظورتون نشدم. یعنی چطوری؟ یعنی به عنوان یک ساختار جدید پست؟

**Comment by Mohammadgeek** [comment]
سلام بله یک ساختار جدید به این شکل که هر کد جالبی که برخوردیم به همراه ریفرنس بزاریم اینجا که بقیه مشاهده کنن یکجور trick
